src/pybind/mgr/rook/module.py

```diff
@@ -445,7 +445,9 @@ def apply_mon(self, spec):
     @handle_orch_error
     def apply_mds(self, spec):
         # type: (ServiceSpec) -> str
-        return self.rook_cluster.apply_filesystem(spec)
+        num_of_osds = self.get_ceph_option('osd_pool_default_size')
+        assert type(num_of_osds) is int
+        return self.rook_cluster.apply_filesystem(spec, num_of_osds)
```

Comment:
since this is the only caller, let's put the get_ceph_option() call inside apply_filesystem() (so that apply_mds doesn't change)

Reply:
Oh i see.. the rook_cluster object doesn't have a MgrModule ref.  Let's go back to passing in the num_replicas value (sorry!)